src/util/date-input-helper.ts

```diff
@@ -31,4 +28,14 @@ export namespace DateInputHelper {
 
         return false;
     };
+
+    export const sanitizeInput = (date: string): string => {
```

Comment:
this helps to convert invalid date strings like "2023-x-x" to an empty string (i.e. those values get ignored)